x-pack/plugins/ml/server/models/job_service/jobs.ts

```diff
@@ -319,12 +319,20 @@ export function jobsProvider(
     if (jobResults && jobResults.jobs) {
       const job = jobResults.jobs.find((j) => j.job_id === jobId);
       if (job) {
+        removeUnClonableCustomSettings(job);
         result.job = job;
       }
     }
     return result;
   }
 
+  function removeUnClonableCustomSettings(job: Job) {
```

Comment:
I'm not sure where this sits in the code, but we're already removing `created_by` on clone and it looks like we're doing at a different place. Might be good to have all job config modifications during clone in the same place?

Reply:
These properties should be removed by elasticsearch when getting the job and supplying `exclude_generated`
For now we can remove them here and I indent to raise an issue for this change in elasticsearch.